instrumentation/all/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-all
 
+### v0.63.0 / 2024-08-15
+
+* (No significant changes)
```

Comment:
I think we should mention something here. Also, the PR description should match this version number.

Reply:
```suggestion
* ADDED: Collect pg db.collection_name attribute
```